### PR TITLE
Shim for install-plugins -> plugin-installation-manager-tool

### PR DESCRIPTION
**Pull request by slide**
This replaces the contents of install-plugins.sh to use plugin-manager-installation-tool internally. Requires an update to plugin-installation-manager-tool 2.5.0 (#1071 does this as well).

### CODE REVIEW

File: install-plugins.sh [hunk 1]
```diff
@@ -12,210 +12,21 @@
 # JENKINS_UC_EXPERIMENTAL: url of the Experimental Update Center for experimental versions of plugins. Default: ""
 # JENKINS_INCREMENTALS_REPO_MIRROR: url of the incrementals repo mirror. Default: ""
 # JENKINS_UC_DOWNLOAD: download url of the Update Center. Default: JENKINS_UC/download
-# CURL_OPTIONS When downloading the plugins with curl. Curl options. Default: -sSfL
-# CURL_CONNECTION_TIMEOUT When downloading the plugins with curl. <seconds> Maximum time allowed for connection. Default: 20
-# CURL_RETRY When downloading the plugins with curl. Retry request if transient problems occur. Default: 3
-# CURL_RETRY_DELAY When downloading the plugins with curl. <seconds> Wait time between retries. Default: 0
-# CURL_RETRY_MAX_TIME When downloading the plugins with curl. <seconds> Retry only within this period. Default: 60
```
[the remark this answers is not in the document]
Comment (timja) [reply install-plugins.sh:19]: https://github.com/jenkinsci/plugin-installation-manager-tool#proxy-support

**Review by garethjevans — APPROVED** [review]

**Comment by slide** [comment]
Do we want to expose the options for proxy stuff via environment variables?

**Comment by slide** [comment]
Is this ok to merge? Any items that people want to see?

@oleg-nenashev @MarkEWaite

**Comment by MarkEWaite** [comment]
> Is this ok to merge? Any items that people want to see?
> 
> @oleg-nenashev @MarkEWaite

[Y.R. Sudhakar](https://www.linkedin.com/in/yrsudhakar/) has proposed a first draft of a [plugin installation manager section](https://groups.google.com/g/jenkinsci-docs/c/q3hFdXPUGLc/m/RHXgUuNZBAAJ) for the ["Managing Plugins" page](https://www.jenkins.io/doc/book/managing/plugins/) of the Jenkins Handbook on https://www.jenkins.io.  I'd prefer that we have that documentation available and a blog post to announce the change before this is merged.

**Comment by ianwilliams1** [comment]
Should updating the [README - Usage](https://github.com/jenkinsci/docker#usage-1) section be included as part of this change or is a issue required?

**Comment by slide** [comment]
I don't know if this will be merged or not, so whatever others decide.

**Comment by dduportal** [comment]
Hello there ! What is the status of this one? That would be really useful but might need a check during the paltform SIG meeting as it might impact end users (but for their own good).

**Comment by timja** [comment]
Should be fine once conflicts are resolved I think

**Comment by dduportal** [comment]
> Should be fine once conflicts are resolved I think

Shouldn't we communicate on this as it might slightly change end users build behaviors?

**Comment by timja** [comment]
> > Should be fine once conflicts are resolved I think
> 
> Shouldn't we communicate on this as it might slightly change end users build behaviors?

could do, do you mean in the README or something else?

**Comment by timja** [comment]
Alternative: https://github.com/jenkinsci/docker/pull/1408

**Pull request closed without merging**